src/workshops/4/pgr.c

```diff
@@ -0,0 +1,53 @@
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+enum { TWO = 2, FIVE = 5, NINE = 9 };
+
+/* Denne funktion skal returnere 1 hvis x er et primtal og 0 ellers */
+int isPrime(int number) {
```

Comment:
@MadsHeilmann https://google.github.io/styleguide/cppguide.html#Function_Names

Reply:
@MadsHeilmann You can return the `bool` data type in C23. https://en.cppreference.com/w/c/language/bool_constant
https://en.cppreference.com/w/c/language/arithmetic_types#Boolean_type